pkg/controllers/state/cluster.go

```diff
@@ -513,6 +513,14 @@ func (c *Cluster) populateInflight(ctx context.Context, n *StateNode) error {
 	if err != nil {
 		return client.IgnoreNotFound(err)
 	}
+	if n.NodeClaim != nil && n.Node != nil {
```

Comment:
I think we can just get away with a condition here that checks if the node is initialized and returns nil to exit if it is. Since the inflightAllocatable is _only_ used if the node isn't initialized, I think that should be sufficient. So something like

```go
if n.Initialized() {
  return nil
}
```